inthe_am/taskmanager/models/taskstore.py

```diff
@@ -288,6 +293,10 @@ def _get_extra_safely(self, key, val):
                 self._is_numeric
             ),
             (
+                re.compile('^uda\.priority\.default$'),
```

Comment:
Not quite sure how to reference it. Can you take a look at commit 151427a88fbf682b8a6160c2a2b80bac452fa793
Is this what you had in mind?